### PR TITLE
Enforce max line length of 160 characters

### PR DESCRIPTION
**Pull request by AlanCoding**

### CODE REVIEW

**Comment by jangsutsr** [comment]
120 might be better? As it would fit in most of the modern monitors.

**Comment by AlanCoding** [comment]
I think 120 would be much more disruptive. There will be a lot to deal with if go and run that, to the point that some code may actually need to be refactored.

**Comment by matburt** [comment]
I guess the better question is, why do we need this? :)

**Comment by AlanCoding** [comment]
> why do we need this? :)

There's a reverse to that question. If someone makes a change that creates a line 2 times longer than any other lines inside of a file with many 1,000s of lines, why should we accept that? Why does someone need to make a line that long? I don't think we should have an oppressively tiny line length limit, but it would be nice to have a limit at _some_ length.

**Comment by matburt** [comment]
I can get on board with that... CONTRIBUTING probably needs to be updated also with some of our style guidelines.

**Comment by matburt** [comment]
maybe resolve the merge conflict now.

File: setup.cfg [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 # W391 - Blank line at end of file
 # W293 - Blank line contains whitespace
-ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,E501,W291,W391,W293
+ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293
 exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory/ec2.py,awx/plugins/inventory/gce.py,awx/plugins/inventory/vmware.py,awx/plugins/inventory/openstack.py,awx/ui,awx/api/urls.py,awx/main/migrations,awx/main/south_migrations,awx/main/tests/data,installer/openshift/settings.py
 
 [flake8]
-ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,E501,W291,W391,W293,E731,F405
+max-line-length=160
+ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293,E731,F405
```
Comment (AlanCoding) [inline setup.cfg:20]: here are the important flake8 rule changes.

**Review by matburt — APPROVED** [review]
lgtm

**Pull request merged**